index.js

```diff
@@ -1 +1,61 @@
-// Do work!
\ No newline at end of file
+// Do work!
+
+// We need to create a function to check the validity of a password. It must have at least 8 characters and must have at least one of each of the following:
+// lower case letter, upper case letter, numeric value, special character. If any of these conditions are not met, the password is invalid.
+
+// Let's start by checking for validity and then move on to false returns.
+
+module.exports = validatePassword
+
+
+function validatePassword(password) {
```

Comment:
I like this breakup of the code. check the length and then we are checking characters. Using the `characterChecker` is excellent work. Nice and readable code.